sdk/nodejs/src/client.ts

```diff
@@ -40,6 +40,10 @@ export class DVCClient {
         this.options = options
         this.logger = options?.logger || dvcDefaultLogger({ level: options?.logLevel })
 
+        if (options?.enableEdgeDB) {
+            this.logger.info('EdgeDB can only be enabled for the DVC Cloud Client.')
```

Comment:
Should this be a warning?

Reply:
I don't believe so, this will not prevent the SDK from functioning correctly... it will simply not leverage the Edge DB functionality, so I made it an info level log